ext/url/config.m4

```diff
@@ -0,0 +1,15 @@
+dnl Configure options
+dnl
+PHP_ARG_ENABLE([url],
+  [for URL support],
+  [AS_HELP_STRING([--enable-url],
+    [Enable URL support])])
+
+if test "$PHP_URL" != "no"; then
+    PHP_LEXBOR_CFLAGS="-I$abs_srcdir/ext/dom/lexbor -DLEXBOR_STATIC"
+
+    AC_DEFINE(HAVE_URL, 1, [Whether the url extension is enabled])
+    PHP_INSTALL_HEADERS([ext/url], [php_url.h])
+    PHP_ADD_EXTENSION_DEP(url, dom)
```

Comment:
wouldn't be better to move lexbor to its own extension? It might be potentially useful as this could be then always enabled and it could also be potentially possible to allow replacing with the distro version if it gets available on distros. Also this would prevent this to have indirect dependency on libxml.

Reply:
Yes, this is the plan with the final code. Until then I didn't want to deal with the additional complexity of creating 2 extensions.